serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc

```diff
@@ -210,6 +213,13 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+[NOTE]
+====
+The Job Service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be
+switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option.
+More details about the Job Service Container image can be found link:{jobs_service_image_usage_url}(here).
```

Comment:
So I went to the link to see what the other options are and found this:
https://github.com/kiegroup/kogito-images/blob/main/kogito-jobs-service-allinone-overrides.yaml#L25-L27
`The all in one image contains all the supported jdbc plugins for Jobs Service, ephemeral, infinispan, mongodb and infinispan, if empty, defaults to ephemeral`
So the default is ephemeral according to the description and there are also infinispan and mongodb options to choose from.

Reply:
correct, but, AFAIK for serverless only postgresql and inmemory are supported at this time

Reply:
That may be true but we still say it defaults to `inmemory` whereas in the page it says it defaults to `ephemeral`.